src/components/ha-selector/ha-selector-select.ts

```diff
@@ -273,13 +279,16 @@ export class HaSelectSelector extends LitElement {
     });
   }
 
+  private _openedChanged(ev?: CustomEvent): void {
+    if (ev?.detail.value) {
+      this._filterChanged();
```

Comment:
this would clear the filter when you open it

Reply:
It is not a problem because it's already deleted when `_comboBoxValueChanged`.
1. For multiple: true. The field is always empty because selected items are displayed as chip
2. For multiple: false. The value is stored in `this.value`, not `this._filter`